explainability-service/src/main/java/org/kie/trustyai/service/payloads/service/ServiceMetricsScheduledMetadata.java

```diff
@@ -1,10 +1,23 @@
 package org.kie.trustyai.service.payloads.service;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class ServiceMetricsScheduledMetadata {
-    public int spd = 0;
-    public int dir = 0;
+    private Map<String, Integer> metricCounts = new HashMap<>();
```

Comment:
I'm not sure to understand the goal of this map, what is it going to contain?

Reply:
This contains the total number of requested metrics per type. So the original was 

```
int spd=0
int dir=0
```

the map equivalent would be `{"spd":0, "dir": 0}`